modules/security-context-constraints-requiring.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-requiring_{context}"]
+= Configuring a workload to require a specific SCC
+
+You can configure a workload to require a certain security context constraint (SCC). This is useful in scenarios where you want to pin to a specific SCC to the workload or if you want to prevent your required SCC from being preempted by another SCC in the cluster.
+
+To require a specific SCC, set the `openshift.io/required-scc` annotation on your workload. You can set this annotation on any resource that can set a pod manifest template, such as a deployment or daemon set.
+
+The SCC must already exist in the cluster and must be applicable to the workload, otherwise pod admission will fail.
+
+[WARNING]
+====
+Do not change the `openshift.io/required-scc` annotation in the live pod's manifest; the pod admission will fail.
+====
+
+.Prerequisites
+
+// TODO: Verify that there is no case where you have to be the cluster admin here
```

Comment:
@s-urbaniak Just confirming that you don't have to be cluster-admin to do anything related to setting the required SCC. Not even if it's a highly privileged SCC they're trying to use?

Reply:
@bergerhoffer the actual prerequisite is denoted in line 13:
```
The SCC must already exist in the cluster and must be applicable to the workload, otherwise pod admission will fail.
```
Maybe worth putting it here?

Reply:
Yep good suggestion, can do. Nothing requiring cluster-admin though that we'd need to note?

Reply:
Actually, I do already list that as a prerequisite below:

```
* The SCC must already exist in the cluster.
```

Reply:
> Nothing requiring cluster-admin though that we'd need to note?

no, the only permission the "user" needs to have is to set annotations on its own deployment manifests.

Having said that, the SCC must be **applicable** to the workload. I describe what applicable means in the original PR https://github.com/openshift/openshift-docs/pull/62558/files:
```
An SCC is considered applicable if the requesting user creating the pod or the pod's service account
have `use` permissions for the required SCC in the requested pod's namespace.
```